test/expr/unary/selector/selector.swift

```diff
@@ -146,3 +146,19 @@ case #selector(SR1827.bar)?:
 default:
   break
 }
+
+// SR-3174
+
+class Hello {
```

Comment:
In addition to these tests, please add one with nested declarations and another that uses a top-level selector.